runtime/doc/develop.txt

```diff
@@ -322,6 +322,11 @@ General rule: One statement per line.
 
 Wrong:	    if (cond) a = 1;
 
+Wrong:	    if (cond)
+	    {
+		a = 1;
+	    }
```

Comment:
This part is the example for the rule "One statement per line."
So, adding here is not good.

BTW, L358 says "When a block has one line the braces *can* be left out."
It's not necessary to remove braces. Also `:help C99` says:

```
Please don't make changes everywhere to use the C99 features, it causes merge
problems for existing patches.  Only use them for new and changed code.
```

I think it is the same for this.

Reply:
> So, adding here is not good.

Indeed. I remove this.

> I think it is the same for this.

I understood what you are saying. However, since I started working, I did it to the end.
I will not work any more. Leave it to Bram to merge or not. :smile: